coalib/output/Logging.py

```diff
@@ -1,10 +1,26 @@
 from datetime import datetime
+from collections import defaultdict
 import json
 import io
 import logging
 import logging.config
 
 
+class CounterHandler(logging.Handler):
+    """
+    A logging handler which counts the number of calls
+    for each logging level.
+    """
+    level_called = None
+
+    def __init__(self, *args, **kwargs):
+        super(CounterHandler, self).__init__(*args, **kwargs)
+        CounterHandler.level_called = defaultdict(lambda: 0)
```

Comment:
The cleanest thing to do would be to have it as a regular instance member. But as I pointed it's a bit hard to access the handlers via the logger. I'm gonna go into a bit of detail, maybe you have a better idea :smile: 

So in `coala_main` I could get the logger and then access the list of handlers in `logger.handlers`. I could loop through it and select the one that is an instance of `CounterHandler`. But that's kind of awkward to do and it creates a problem with branch coverage, as I would need to add a test for the case where the CounterHandler isn't there, which should never be the case. 

Hope that's not too confusing. Would appreciate any ideas :)